### DISCUSSION TITLE
XYZPDB incorrect format conversion.

### DISCUSSION THREAD

**Discussion by PinkFoxK** [TinkerTools/tinker#136] [Q&A] [answered]
We tried to convert the .xyz file to .pdb (force field - amoebabio18). When using the xyzpdb utility, the output file of the .pdb format is incorrect. The types of atoms do not correspond to the generally accepted nomenclature, there are no names of amino acid residues. The column with the type of atoms is stored in the pdb file (attached in the file with the force field). Has anyone encountered a similar problem? What other scripts can be used to convert formats (with unlinking the force field file)?

**Comment by jayponder** [comment, marked as answer]
Hi, In order use XYZPDB to convert a Tinker XYZ file to PDB format with the correct residue names, etc., you must have a file with the biopolymer sequence in Tinker format, a ".seq" file. If the sequence file is not present, then XYZPDB will create a generic PDB file without any of the usual PDB features.

The ".seq" files are created by Tinker when you go in the other direction- from PDB to XYZ using the PDBXYZ program. A ".seq" is also created when you build a PDB file from sequence using the Tinker PROTEIN or NUCLEIC programs. Also, note the format of the ".seq" file is very simple, and you can fairly easily create one yourself if you know the sequence of your biopolymer. Example ".seq" files can be found in the /example directory of the Tinker distribution.
